_layouts/default.html

```diff
@@ -4,136 +4,156 @@
 
 <!DOCTYPE html>
 
-<html lang="{{ site.lang | default: "en-US" }}">
+<html lang="{{ site.lang | default: 'en-US' }}">
 {% include head.html %}
 <body>
   <svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
-    <symbol id="link" viewBox="0 0 16 16">
+    <symbol id="svg-link" viewBox="0 0 16 16">
       <title>Link</title>
       <path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path>
     </symbol>
+    <symbol id="svg-search" viewBox="0 0 24 24">
```

Comment:
Where did these icons come from? I'd love to give the author credit in our docs.

Reply:
these are material icons
https://material.io/resources/icons/?icon=search&style=baseline
https://material.io/resources/icons/?icon=menu&style=baseline
https://material.io/resources/icons/?icon=navigate_next&style=baseline
https://material.io/resources/icons/?search=file&icon=insert_drive_file&style=outline

Reply:
Do you think we could replace them with: https://feathericons.com? I'd love to support an indie developer/designer.

Reply:
Of course

Reply:
@pmarsceill Replaced the icons with similar ones from https://feathericons.com
The icons look a little bit different but very similar

Reply:
I used these:
https://feathericons.com/?query=search
https://feathericons.com/?query=menu
https://feathericons.com/?query=link
https://feathericons.com/?query=chevron-right
https://feathericons.com/?query=file

Reply:
Thank you! This looks great.